salesforce/management/commands/upload_mapbox_schools.py

```diff
@@ -30,7 +30,7 @@ def handle(self, *args, **options):
 
         for school in schools:
             total_schools = total_schools + 1
-            if school.lat and school.long:
+            if (school.lat and school.long) and (school.lat != 0 and school.long != 0):
```

Comment:
Is it wrong to have _either one_ be zero, or only wrong if they _both_ are?

Reply:
@RoyEJohnson - Ah, you're right, this should be an `or` statement!